packages/ec-linear/examples/count_ones/main.rs

```diff
@@ -64,7 +64,7 @@ fn main() -> Result<()> {
 
     let num_test_cases = bit_length;
 
-    let selector = Weighted::new(Best, 1)
+    let selector = DynWeighted::new(Best, 1)
```

Comment:
Question: Do we want `DynWeighted` or just `Weighted` in the examples?

Reply:
We'll address this in #263.